utils/pytorch.py

```diff
@@ -38,21 +40,22 @@ def __init__(self, model, disable_jit_freeze=False, example_inputs=None, func=No
         else:
             cached_dir = Path(os.path.dirname(os.path.realpath(__file__)) + "/cached")
             cached_path = cached_dir / f"{self.__model._get_name()}_{hashlib.sha224(str(model).encode('utf-8')).hexdigest()}.pt"
-            if cached_path.exists():
+            if version.parse(pkg_resources.get_distribution("torch").version) >= version.parse("2.0"):
+                self.__frozen_script = torch.compile(self.__model)
```

Comment:
Good question is whether torch.compile does offer on-par performance with torch.jit compilation and should be our preferred method in all cases. I would say for now let's use it as a back-up when torch.jit doesn't work, i.e. as a second try-except clause catching torch.jit.trace errors. After we have support in AIO we will be able to further investigate and establish what should be the default behavior.